pkg/kvstore/consul.go

```diff
@@ -433,7 +434,7 @@ func (c *ConsulClient) Watch(w *Watcher, list bool) {
 						Value: newPair.Value,
 					}
 				} else {
-					log.Warning("consul: Previously unknown key %s received with CreateIndex(%d) != ModifyIndex(%d)",
+					log.Warn("consul: Previously unknown key %s received with CreateIndex(%d) != ModifyIndex(%d)",
```

Comment:
Update the log message to indicate what is being done about the issue:
`"consul: [...] ModifyIndex(%d); ignoring update"`